protobuf-kafkaconnect-converter/src/test/proto/EnumTypeSyntax2.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto2";
+
+package com.amazonaws.services.schemaregistry.kafkaconnect.tests.syntax2;
+
+message EnumTest {
+  enum Corpus {
+    UNIVERSAL = 0;
+    WEB = 1;
+    IMAGES = 2;
+    LOCAL = 3;
+    NEWS = 4;
+    PRODUCTS = 5;
+    VIDEO = 6;
+  }
+  required Corpus corpus = 1 [default = IMAGES];
```

Comment:
How is default value reflected in the connect data?

Reply:
Not sure how the default value is supposed to work, but it seems like it isn't necessary for anything as it runs fine with it removed. I will leave it out.